Resources/Prototypes/_NF/Entities/Mobs/Species/goblin.yml

```diff
@@ -0,0 +1,102 @@
+- type: entity
+  save: false
+  parent: BaseMobSpeciesOrganic
+  name: Urist McGoblin
+  id: MobGoblinBase
+  abstract: true
+  components:
+  - type: Hunger
+    starvationDamage:
+      types:
+        Cold: 0.1
+        Bloodloss: 0.1
+  - type: Thirst
+  - type: Carriable # Carrying system from nyanotrasen.
+  - type: Stamina
+  - type: PseudoItem
+  - type: Perishable
+  - type: Respirator
+    damage:
+      types:
+        Asphyxiation: 2
+    damageRecovery:
+      types:
+        Asphyxiation: -1.0
+  - type: Butcherable
+    butcheringType: Spike
+    spawned:
+    - id: FoodMeatGoblin
+      amount: 3
+  - type: Icon # It will not have an icon in the adminspawn menu without this. Body parts seem fine for whatever reason.
+    sprite: _NF/Mobs/Species/Goblin/parts.rsi
+    state: full
+  - type: Sprite
+    scale: 0.8, 0.7
+  - type: HumanoidAppearance
+    species: Goblin
+  - type: Fixtures
+    fixtures: # TODO: This needs a second fixture just for mob collisions.
+      fix1:
+        shape:
+          !type:PhysShapeCircle
+          radius: 0.25
+        density: 135
+        restitution: 0.0
+        mask:
+        - MobMask
+        layer:
+        - MobLayer
+  - type: Body
+    prototype: Goblin
+    requiredLegs: 2
+  - type: Damageable
+    damageModifierSet: Goblin
+  - type: MeleeWeapon
+    soundHit:
+      collection: Punch
+    animation: WeaponArcClaw
+    damage:
+      types:
+        Slash: 4
```

Comment:
4 Slash is still pretty strong, its causes bleeding.

Reply:
Can lower eve further, to 3 for example